### ISSUE TITLE
Default to Drawer open

### ISSUE THREAD

**Issue by Layla-P** [mac81/pure-drawer#29]
Hi, great product, best I have seen.  However I'm trying to figure out how to default it to open?
I need it to be open for desktops and closed for mobile devices.  Any help you can give would be greatly appreciated.

**Comment by mac81** [comment]
Hi

Thanks for trying out Pure Drawer.

To have the drawer default open is as simple as putting a checked="true" attribute on the .pure-toggle input.

Making it switch state between open and closed on mobile and desktop i a bit trickier as you will need to use javascript.

On the other side. If the drawer will always have an open state on desktop you can get around it with just tweaking some css. (Toggleable on mobile, always open on desktop).

Please let me know which of these functionalities you would prefer and I'l try and sort it out for you. 
And if you use the sass or css version of pure drawer.

**Comment by Layla-P** [comment]
Thank you so much for your super quick response.  I thought it would be something quick and easy :-)
I have tried out several js drawers and none of them worked as I wished.
I tried yours last week and it's been super!  All I had to tweak was the addition of a scrollbar, I'm using pure-effect-fade.
I reckon the  (Toggleable on mobile, always open on desktop) option would be great.  I think material design lite's drawer has an option similar to this.
I will have a play!  Keep up the awesome work!
